client/app/telecom/telephony/alias/configuration/telecom-telephony-alias-configuration.js

```diff
@@ -5,8 +5,25 @@ angular.module('managerApp').config(($stateProvider) => {
       'aliasInnerView@telecom.telephony.alias': {
         templateUrl: 'app/telecom/telephony/alias/configuration/telecom-telephony-alias-configuration.html',
         controller: 'TelecomTelephonyAliasConfigurationCtrl',
-        controllerAs: 'AliasConfigurationCtrl',
+        controllerAs: '$ctrl',
       },
+      'featureView@telecom.telephony.alias.configuration': {
+        templateProvider: alias => `<div data-ng-include="::'app/telecom/telephony/alias/configuration/feature/${alias.featureType}/telecom-telephony-alias-configuration-${alias.featureType}.html'"></div>`,
+        controllerProvider: (alias) => {
```

Comment:
I know I've could destructure the param, but for readability, I've let the param this way